### DISCUSSION TITLE
Kastrull suggestions

### DISCUSSION THREAD

**Discussion by Smau35** [FakeFishGames/Barotrauma#11695] [Ideas and suggestions]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Kastrull has a couple things in its design that could be changed to improve it:

Kastrull could use another heavy hardpoint as it gets outdone in them by subs that aren't even attack class (Winterhalter, Berilia).
The ventral hardpoint in the middle of the sub could be changed to a heavy one.
![Näyttökuva 2023-03-27 172801](https://user-images.githubusercontent.com/86321358/228011339-f70eed0d-77d2-4d14-ae5c-91c0c08ff09c.png)

Kastrulls descent speed could be improved to around 10 km/h by filling the empty gaps between the ballasts.
![Näyttökuva 2023-03-27 171925](https://user-images.githubusercontent.com/86321358/228011699-e65b59ad-8090-458d-80a7-c8d6a025876b.png)
↓
![Näyttökuva 2023-03-27 172535](https://user-images.githubusercontent.com/86321358/228011985-2c8b532c-b372-4536-9243-adb309566050.png)

### Reproduction steps

-

### Bug prevalence

Happens every time I play

### Version

v1.0.8.0

### Which operating system did you encounter this bug on?

Windows

**Comment by itchyOwl** [comment]
Hi, thanks for your feedback! Next time, please use the [discussions](https://github.com/Regalis11/Barotrauma/discussions) for suggestions. We have so many issue tickets (and more coming in all the time) that we can't accept suggestions here. Let's make this an exception though, so that you don't have to repost it. Maybe someone even does something on it eventually, but no promises.
